internal/grpcwrapper/rawtopic/rawtopicreader/rawtopicreader.go

```
@@ -31,6 +31,7 @@ func (s StreamReader) CloseSend() error {
 func (s StreamReader) Recv() (ServerMessage, error) {
 	grpcMess, err := s.Stream.Recv()
 	if err != nil {
```

```suggestion
	if xerrors.HideEOF(err) != nil {
```

---

What about hide error on err check level?